content/docs/command-reference/metrics/diff.md

```diff
@@ -68,53 +67,40 @@ They're calculated between two commits (hash, branch, tag, or any
 
 ## Examples
 
-Let's employ a simple <abbr>workspace</abbr> with some data, code, ML models,
-pipeline stages, such as the <abbr>DVC project</abbr> created in our
-[Get Started](/doc/tutorials/get-started) section. Then we can see what happens
-with `dvc install` in different situations.
-
-<details>
-
-### Click and expand to setup the project
-
-Start by cloning our example repo if you don't already have it:
+Start by creating a simple metrics file and commit it:
 
 ```dvc
-$ git clone https://github.com/iterative/example-get-started
-$ cd example-get-started
+$ dvc run -M metrics.json \
+        'echo {\"AUC\": 0.9643, \"TP\": 527} > metrics.json'
+$ git add metrics.json metrics.json.dvc
+$ git commit -m "Add metrics file"
 ```
 
-</details>
-
-Notice that we have an `auc.metric` metric file:
-
 ```
-$ cat auc.metric
-0.602818
+$ cat metrics.json
+{"AUC":0.9643, "TP":527}
```

Comment:
2 questions:
- Should the Get Started and its repo be updated to produce JSON metrics?
- What shuold/would happen with `dvc metrics diff` if you have a raw metric first and then update the same file with a JSON (or other format) metric? Probably worth testing.

Reply:
1. Yes, ideally we should update the getting started and remove all the raw metrics and `xpath`s (expect `metrics show`).
2. Good point. It should be the same as a regular text file that was updated to json metrics file.